debian/control.top.in

```diff
@@ -23,10 +23,12 @@ Build-Depends:
     intltool,
     libboost-python-dev,
     libepoxy-dev,
+    libghc-glut-dev,
```

Comment:
That is for the haskell environment. I'd opt for libglut-dev, like
```
libglut-dev|freeglut3-dev
```
which should then work for both bullseye and bookworm (the latter still missing that symbolic link, but we'll eventually get there).

Reply:
`libglut-dev` is not available in the Bullseye sources, but  `freeglut3` or `freeglut3-dev` do work as well.
So 
```diff
- libghc-glut-dev,
+ freeglut3|freeglut3-dev
```
 ?

Reply:
freeglut3-dev is fine for bullseye but for bookworm this is only transitional, libglut-dev is what we should use for the latter, so `libglut-dev|freeglut3-dev` is fine.